cmd/gardener-controller-manager/app/gardener_controller_manager.go

```diff
@@ -235,8 +235,11 @@ func NewGardener(config *componentconfig.ControllerManagerConfiguration) (*Garde
 	logger := logger.NewLogger(config.LogLevel)
 	logger.Info("Starting Gardener controller manager...")
 	logger.Infof("Feature Gates: %s", gardenerfeatures.ControllerFeatureGate.String())
-	if err := flag.Lookup("v").Value.Set(fmt.Sprintf("%d", config.KubernetesLogLevel)); err != nil {
-		return nil, err
+
+	if flag := flag.Lookup("v"); flag != nil {
```

Comment:
We should add the default go flagset to our command's flagsets, then we get this stuff for free - We can do this in this PR, however I'm also fine with this solution.

Reply:
Let's do that separately.